bci_tester/data.py

```diff
@@ -354,7 +354,7 @@ def create_BCI(
         image_type="dockerfile",
         available_versions=["15.4"],
     )
-    for rust_version in ("1.64", "1.65")
+    for rust_version in ("1.64", "1.65", "1.66")
 ]
 (RUST_1_64_CONTAINER, RUST_1_65_CONTAINER) = RUST_CONTAINERS
```

Comment:
```suggestion
(RUST_1_64_CONTAINER, RUST_1_65_CONTAINER, RUST_1_66_CONTAINER) = RUST_CONTAINERS
```

And you need to add RUST_1_66_CONTAINER further below as well (and maybe to the metadata tests as well).

Reply:
Amended. metadata uses the anchor `RUST_CONTAINERS`